catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/MainMenuFragment.java

```diff
@@ -165,6 +165,11 @@ public void onItemClick(@ButtonId int id) {
 		}
 	}
 
+	@Override
+	public void onActivityResult(int requestCode, int resultCode, Intent data) {
+		super.onActivityResult(requestCode, resultCode, data);
```

Comment:
Seems unnecessary.

Reply:
well, sure does seem unnecessary, however it's there to remind me that I should do something with the result. I actually just opened the pr because I wanted to test it and because this should be finished asap.

Reply:
Oh sorry - I forgot to look at Jira first ;-)

Reply:
no, don't worry about it, I shouldn't really have opened it yet 😉